keps/sig-cloud-provider/providers/2020-cloud-provider-inspur.md

```diff
@@ -0,0 +1,145 @@
+IOP has been using this project for several releases:
+- v1.15 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.15/INSPUR-IOP)
+- v1.13 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.13/INSPUR-IOP)
+- v1.11 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.11/INSPUR-IOP)
+- v1.9 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.9/INSPUR-IOP)
+
+As well as IOP, another certified Kubernetes product which used this project is `INSPUR IOP`, and you can refer to 
+following conformance test reports:
+- v1.17 [Conformance Test Report for amd64](https://github.com/cncf/k8s-conformance/blob/master/v1.17/inspur-iop-amd64)
+- v1.17 [Conformance Test Report for arm64](https://github.com/cncf/k8s-conformance/blob/master/v1.17/inspur-iop-arm64)
+- v1.16 [Conformance Test Report for amd64](https://github.com/cncf/k8s-conformance/blob/master/v1.16/inspur-iop-amd64)
+- v1.16 [Conformance Test Report for arm64](https://github.com/cncf/k8s-conformance/blob/master/v1.16/inspur-iop-arm64)
+- v1.15 [Conformance Test Report for amd64](https://github.com/cncf/k8s-conformance/blob/master/v1.15/inspur-iop-amd64)
+- v1.15 [Conformance Test Report for arm64](https://github.com/cncf/k8s-conformance/blob/master/v1.15/inspur-iop-arm64)
+- v1.14 [Conformance Test Report for amd64](https://github.com/cncf/k8s-conformance/blob/master/v1.14/inspur-iop-amd64)
+- v1.14 [Conformance Test Report for arm64](https://github.com/cncf/k8s-conformance/blob/master/v1.14/inspur-iop-arm64)
+
+Other usage of this project can be seen from [GitHub issues](https://github.com/OpenInspur/cloud-provider-inspur/issues).
+
+### Testgrid Integration
+
+INSPUR cloud provider is reporting conformance test results to TestGrid as per the [Reporting Conformance Test Results to Testgrid KEP](https://github.com/kubernetes/enhancements/blob/6427a0becff459815e0e41f72f65ab5f3b8e9c6d/keps/sig-cloud-provider/0018-testgrid-conformance-e2e.md).
+You can get the result from [TestGrid INSPUR Cloud Dashboard](https://testgrid.k8s.io/conformance-cloud-provider-INSPURcloud). 
+
+### CNCF Certified Kubernetes
+
+INSPUR cloud provider is accepted as part of the [Certified Kubernetes Conformance Program](https://github.com/cncf/k8s-conformance).
+
+The conformance test report can be seen from above section `Inspur Open Platform (IOP)` and `INSPUR`. 
+
+### Documentation
+
+INSPUR cloud provider provides [multiple documentation](https://github.com/OpenInspur/cloud-provider-inspur/blob/master/docs) for users as per the [cloud provider documentation KEP](https://github.com/kubernetes/enhancements/blob/6427a0becff459815e0e41f72f65ab5f3b8e9c6d/keps/sig-cloud-provider/20180731-cloud-provider-docs.md).
```

Comment:
Your docs page seems to be 404ing.

Reply:
Could you please use https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/20180731-cloud-provider-docs.md for the cloud-provider-docs link?